operator-demo/README_CN.md

```diff
@@ -22,8 +22,8 @@ $ kubectl create -f config/samples/crd_custom.emqx.io_brokers.yaml
 1. 部署 `operator deployment` 运行
 编译及制作镜像，并推送到镜像仓库
 ```
-$ make docker build IMG=
-$ make docker push IMG=
+$ IMG=xxx make docker-build
+$ IMG=xxx make docker-push
```

Comment:
Do we have a suggested image name for the following deployment？ or is it user-definable?

Reply:
The variable of IMG is user-defineable, such as docker.io/emqx-operator:v0.1.0